base/markdown/GitHub/GitHub.jl

```diff
@@ -33,7 +33,83 @@ function github_paragraph(stream::IO, md::MD, config::Config)
     return true
 end
 
-# TODO: tables
+typealias Row Vector
 
-@flavor github [list, indentcode, blockquote, fencedcode, hashheader, github_paragraph,
-                 en_dash, inline_code, asterisk_bold, asterisk_italic, image, link]
+type Table
+    rows::Vector{Row}
+    align
+end
+
+function github_table(stream::IO, md::MD, config::Config)
+    withstream(stream) do
+        rows = Row[]
+        n = 0
+        align = :r # default is to align right
```

Comment:
Let's go with two spaces for now and if people complain we can do something more elaborate. e.g.

```
+----+-----+
|col | col2|
+----+-----+
|1   |2    |
+----+-----+
```

One thing that I wanted initially was to have color (yellow), however one formatting e.g. code obliterates the formatting, you can see this with 

    md"# Bold `blue` no longer bold"